src/commands/circleci/envvar/update.ts

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { EOL } from 'os';
+import { fstat } from 'fs';
+import { flags, FlagsConfig, SfdxCommand } from '@salesforce/command';
+import { Messages, SfdxError } from '@salesforce/core';
+import { isArray, isString, Dictionary } from '@salesforce/ts-types';
+import { env } from '@salesforce/kit';
+import got from 'got';
+import { green, red, bold, cyan, yellow } from 'chalk';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-release-management', 'circleci');
+
+const URL_BASE = 'https://circleci.com/api/v2/project';
+
+export type EnvvarUpdateStatus = {
+  name: string;
+  message?: string;
+  success: boolean;
+};
+
+export type CircelCIEnvvarUpdateStatus = Dictionary<string | EnvvarUpdateStatus[]>;
+
+export default class CircelCIEnvvarUpdate extends SfdxCommand {
+  public static readonly description = messages.getMessage('envvar.update.description');
+  public static readonly examples = messages.getMessage('envvar.update.examples').split(EOL);
+
+  public static readonly flagsConfig: FlagsConfig = {
+    slug: flags.string({
+      description: messages.getMessage('envvar.update.flags.slug'),
+      char: 's',
+      multiple: true,
+    }),
+    envvar: flags.string({
+      description: messages.getMessage('envvar.update.flags.envvar'),
+      char: 'e',
+      required: true,
+      multiple: true,
+    }),
+    dryrun: flags.boolean({
+      description: 's',
```

Comment:
I assume we want a more thorough description here?

Reply:
I said, "I'll get to that later" I did. Woops.